### DISCUSSION TITLE
Every page in the docs show the same content that is written in next.mdx

### DISCUSSION THREAD

**Discussion by bleedingAyush** [fuma-nama/fumadocs#693] [Q&A] [answered]
Hi, I am using Next 14 and Tailwind in my project. For some reason the test.mdx is showing the same content as index.mdx

https://github.com/user-attachments/assets/3eb6a8cd-1cde-4940-88e2-b84e05194f39

My file structure is like this
-- src/
      -- app/
            --content/
                  index.mdx
                  test.mdx
                  
This is my index.mdx 
```
---
title: Hello World
description: Your first document
---

Welcome to the docs! You can start writing documents in `/content/docs`.

## What is Next?

<Cards>
  <Card title="Learn more about Next.js" href="https://nextjs.org/docs" />
  <Card title="Learn more about Fumadocs" href="https://fumadocs.vercel.app" />
</Cards>

```
This is my test.mdx
```
---
title: Components
description: Components hai bahi
---

## Code Block

```

**Comment by fuma-nama** [comment]
Cannot reproduce on CodeSandbox.

Please provide a reproducible repo/example, and open a bug report with sufficient info for debugging instead.

**Comment by bleedingAyush** [comment]
Thanks for the reply, 
Here is my repo https://github.com/bleedingAyush/Spiralsaas
Here is my site: https://spiralsaas.dev/docs

**Comment by fuma-nama** [comment, marked as answer]
You named the `[[...slugs]]` parameter `slug` in your page.tsx

**Reply by bleedingAyush** [reply]
Thanks actually I copied it from your docs
